sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/E2eTestBase.cs

```diff
@@ -131,9 +131,9 @@ protected async Task<TimeSeriesId> GetUniqueTimeSeriesInstanceIdAsync(TimeSeries
             throw new Exception($"Unique Id could not be found");
         }
 
-        protected async Task<string> getDefaultTypeIdAsync(TimeSeriesInsightsClient client)
+        protected async Task<string> getDefaultTypeIdAsync(TimeSeriesInsightsModelSettings modelSettingsClient)
         {
-            Response<TimeSeriesModelSettings> currentSettings = await client.ModelSettings.GetAsync().ConfigureAwait(false);
+            Response<TimeSeriesModelSettings> currentSettings = await modelSettingsClient.GetAsync();
```

Comment:
was removing ConfigureAwait intenional?

Reply:
It was but then realized in samples we don't use it, tests still do. Addressed it now.